GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -141,16 +141,16 @@ public FormBrowse([CanBeNull] GitUICommands commands, string filter, ObjectId se
                 ImageSize = DpiUtil.Scale(new Size(16, 16)),
                 Images =
                 {
-                    Images.CommitSummary,
-                    Images.FileTree,
-                    Images.Diff,
-                    Images.Key
+                    { nameof(Images.CommitSummary), Images.CommitSummary },
+                    { nameof(Images.FileTree), Images.FileTree },
+                    { nameof(Images.Diff), Images.Diff },
+                    { nameof(Images.Key), Images.Key }
                 }
             };
-            CommitInfoTabControl.TabPages[0].ImageIndex = 0;
```

Comment:
This (or previous comit) changes the default taborder, It is Commit-FileTree-Diff when no Commit left panel
OK change for me though

Reply:
Adding a button to easily toggle the position of the "commit info" pane uncovered a but where the "file tree" and "diff" tabs could become transposed. Previously you had to go to settings to change this, so I doubt many/any saw this bug, and it would be fixed on restart.

That tab ordering actually felt more logical that way to me. The diff feels more closely linked to the commit that the tree, at least to me. Perhaps it was this way in the past because the diff tab item changed its length, and would reposition tabs to its right?

![image](https://user-images.githubusercontent.com/350947/42811547-ca6766dc-89b2-11e8-964b-9b78c0e92692.png)

It's easy enough to change back if people dislike the change.

Reply:
I suggest to keep it (I prefer it), just noting the change

Reply:
LGTM